MonoGame.Framework/Platform/Graphics/OpenGL.iOS.cs

```diff
@@ -35,7 +35,7 @@ public class GraphicsContext : IGraphicsContext
     {
         public GraphicsContext ()
         {
-            Context = new EAGLContext (EAGLRenderingAPI.OpenGLES2);
+            Context = new EAGLContext (EAGLRenderingAPI.OpenGLES3);
```

Comment:
Documentation is sparse, so I don't know what happens here if the device does not support GLES3. I assume it will either throw or return null. I don't see an easy way to fallback to GLES2 in the API. I think we need to create a context and use `glGetString` to check the supported version if we want to continue to support GLES2.

Not sure if we *should* still support GLES2 on iOS though. A quick search indicates iPhone supported GLES3 in iOS 7+ since iPhone 5S/iPad Air.

Reply:
Yeah, iOS devices have  good upgrade rate to latest version.

Reply:
Yeah i think for now it would be best to fallback to GLES2 if possible.  It would suck to hurt install base of existing users unless we really have to.